packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -597,6 +634,149 @@ export const prepareVaultManagerKit = (
           return E(metricsTopicKit.recorder).write(payload);
         },
 
+        /**
+         * @param {TimestampRecord} timestamp
+         * @returns {Promise<LiquidationVisibilityWriters>}
+         */
+        async makeLiquidationVisibilityWriters(timestamp) {
+          const liquidationRecorderKits =
+            await this.facets.helper.makeLiquidationRecorderKits(timestamp);
+
+          /** @param {VaultData} vaultData */
+          const writePreAuction = vaultData => {
+            /** @type PreAuctionState */
+            const preAuctionState = [...vaultData.entries()].map(
+              ([vault, data]) => [
+                `vault${vault.getVaultState().idInManager}`,
+                { ...data },
+              ],
+            );
+
+            return E(
+              liquidationRecorderKits.preAuctionRecorderKit.recorder,
+            ).writeFinal(preAuctionState);
+          };
+
+          /**
+           * @param {PostAuctionParams} params
+           * @returns {Promise<void>}
+           */
+          const writePostAuction = ({ plan, vaultsInPlan }) => {
+            /** @type PostAuctionState */
+            const postAuctionState = plan.transfersToVault.map(
+              ([id, transfer]) => [
+                `vault${vaultsInPlan[id].getVaultState().idInManager}`,
+                {
+                  ...transfer,
+                  phase: vaultsInPlan[id].getVaultState().phase,
+                },
+              ],
+            );
+            return E(
+              liquidationRecorderKits.postAuctionRecorderKit.recorder,
+            ).writeFinal(postAuctionState);
+          };
+
+          /** @param {AuctionResultsParams} params */
+          const writeAuctionResults = ({
+            plan,
+            totalCollateral,
+            totalDebt,
+            auctionSchedule,
+          }) => {
+            /** @type AuctionResultState */
+            const auctionResultState = {
```

Comment:
Please see `Desired data to be exposed on vstorage` section [here](https://github.com/Jorge-Lopes/liquidation-visibility/issues/2#issue-2060228731). We've made our decisions based on that.

Reply:
@anilhelvaci having reviewed these fields - I think it would be a missed opportunity not to expose them in vstorage as visibility on these add value. Apologies for not including them in initial spec.

Specfically `collateralForReserve, debtToBurn,  mintedForReserve` `and totalPenalty.` I think you said in our call on friday that collateralForReserve was already included.

Reply:
> I think you said in our call on friday that collateralForReserve was already included.

Correct.

> Apologies for not including them in initial spec.

No issues at all!

I'll include these fields and update the tests accordingly 👍

Reply:
Here's the issue to track this => https://github.com/Jorge-Lopes/agoric-sdk/issues/36